docs/infoblox-openshift.md

```diff
@@ -101,10 +103,23 @@ Make sure your AWS account is subscribed to the following product:
         - Admin password is better to be reset if you plan to use this Infoblox for a long time
         - Enable NTP if you plan to use this Infoblox for a long time
     - Restart will be needed at the end of the setup of the new grid
-- Start DNS service: `Grid` top tab -> `Grid manager` subtab -> `DNS` -> Select `infoblox.localdomain` (or your instance's FQDN) -> Button `Play` (Start)
-- Add name server group with the grid server: `Data Management` top tab -> `DNS` subtab -> `Add` right panel -> `Group` -> `Authorative` -> Put a name -> `+` button -> `Add Grid Primary` -> `Select` -> `Add` -> `Save & Close`
-- Default self signed certificate uses the private IP, you would need to regenerate it with the Elastic IP:
-    - `Grid` top tab -> `Grid Manager` subtab -> `DNS` -> `Certificates` right panel -> `HTTPS Cert` -> `Generate Self Signed Certificate` -> Add Elastic IP (or FQDN if any) in `Subject Alternative name` and fill `Days Valid` -> Accept the restart of the service
+- Add name server group with the grid server: `Data Management` top tab -> `DNS` subtab -> `Add` dropdown button on the right panel -> `Group` -> `Authorative` -> Put a name (e.g. `infoblox`) -> `+` button -> `Add Grid Primary` pannel -> `Select` button -> `Add` button -> `Save & Close`
+- Start DNS service: `Grid` top tab -> `Grid manager` subtab -> `DNS` check button -> `Services` subsubtab -> Select `infoblox.localdomain` (or your instance's FQDN) -> Button `Play` (Start) -> Confirm the start
+- Default self signed certificate uses the private IP, you would need to regenerate it with the Elastic IP (or FQDN if any):
```

Comment:
*nit* I don't quite understand what this sentence is saying, maybe this is better?

```suggestion
- The default self-signed certificate uses the private IP. Therefore, you will need to regenerate it with the Elastic IP (or FQDN if any):
```